tileview/src/main/java/com/qozix/tileview/tiles/TileCanvasViewGroup.java

```diff
@@ -248,12 +249,13 @@ void onRenderTaskPostExecute() {
     if( mTileRenderListener != null ) {
       mTileRenderListener.onRenderComplete();
     }
-    invalidate();
-    requestRender();
```

Comment:
That requestRender causes an infinite loop (`beginRender` being called, and it is definitely not necessary).